src-plugins/itkFilters/itkMorphologicalFiltersProcessBase_p.h

```diff
@@ -140,16 +155,18 @@ class ITKFILTERSPLUGIN_EXPORT itkMorphologicalFiltersProcessBasePrivate : public
         output->setData ( filter->GetOutput() );
 
         // Add description on output data
-        QString newSeriesDescription = input->metadata ( medMetaDataKeys::SeriesDescription.key() );
+        QString newSeriesDescription = filenameDescription + " ";
 
         if (isRadiusInPixels)
-            newSeriesDescription += description + "\n("+ QString::number(floor(radius[0]))+", "+
-            QString::number(floor(radius[1]))+", "+ QString::number(floor(radius[2]))+" pixels)";
+        {
+            newSeriesDescription += QString::number(floor(radius[0])) + " px";
+        }
         else
-            newSeriesDescription += description + "\n("+ QString::number(radiusMm[0])+", "+
-            QString::number(radiusMm[1])+", "+ QString::number(radiusMm[2])+" mm)";
+        {
+            newSeriesDescription += QString::number(float(radiusMm[0])) + " mm";
```

Comment:
The dilation/erosion kernel is 3dimensional, so you need to display the x, y,z radius.